pkg/controller/controller.go

```diff
@@ -102,8 +106,46 @@ func NewController(clientset kubernetes.Interface, ssclientset ssclientset.Inter
 		},
 	})
 
+	sInformer := sinformer.Core().V1().Secrets().Informer()
+	sInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		DeleteFunc: func(obj interface{}) {
+			skey, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
+			if err != nil {
+				log.Printf("failed to fetch Secret key: %v", err)
+				return
+			}
+
+			ns, name, err := cache.SplitMetaNamespaceKey(skey)
+			if err != nil {
+				log.Printf("failed to split metadatada namespace: %v", err)
+				return
+			}
+
+			ssecret, err := ssclientset.BitnamiV1alpha1().SealedSecrets(ns).Get(context.Background(), name, metav1.GetOptions{})
+			if err != nil {
+				if !strings.Contains(err.Error(), "not found") {
+					log.Printf("failed to find the Sealed Secret associated: %v", err)
```

Comment:
This log is only printed if the error is not "not found", implying something went wrong when getting the SealedSecret. 

```suggestion
					log.Printf("failed to get SealedSecret: %v", err)
```